Sources/TypeAlias.swift

```diff
@@ -5,13 +5,13 @@
 #endif
 
 #if os(OSX)
-  public typealias Controller = NSViewController
+  public typealias CurrentController = NSViewController
```

Comment:
I think we should try and fix that thing, now is probably a good idea to address it as we are already doing some breaking changes in the migration. (breaking as in having to adopt new Swift 3 syntax).